### ISSUE TITLE
This looks real good...

### ISSUE THREAD

**Issue by ryanrca** [galenyuan/vue-datatable#3]
... but how do you use it?  The docs folder just has code.

It needs server requirements and usage examples.  Does it work with vue 2.0?

Thanks,!

**Comment by jpmurray** [comment]
I'll add my voice to it. I'm just browsing to get a vuejs datatable implementation that would work, and yours seems to be one of the few to do that easily, but there's a bit lacking to the docs.

**Comment by galenyuan** [comment]
actually, basic usage is already in **README.md**, doc folder is just for generate a github page
this component not support vue2.0 for now, planning to support it soon.

**Comment by whiteboardmonk** [comment]
Curious to know if this is simply a more convenient replacement for Data-tables in a vue.js project OR using this will be more optimum by possibly taking advantage of virtual dom etc.?

**Comment by galenyuan** [comment]
@whiteboardmonk it's just a component to build a table which have sort, pagination and search in frontend

**Comment by francisortega** [comment]
> a component to build a table which have sort, pagination and search in frontend

So does this also include an editable filter for each column? Thanks in advance.

**Comment by liujinyang1994** [comment]
I can't take it anymore,  this is a god. 

> value: [ text: 'button', class: ['button'] ... ]

Such grammar！

How can you get 66 star?

**Comment by lmj0011** [comment]
>I'll add my voice to it. I'm just browsing to get a vuejs datatable implementation that would work, and yours seems to be one of the few to do that easily, but there's a bit lacking to the docs.

@jpmurray I agree, I tried implementing this datatable: https://github.com/ratiw/vue-table

but failed, because of so much setup just to get the what you see in the example.

I hoping this lib, just works. Like the jQuery version

**Comment by jdriesen** [comment]
@jpmurray  I also tried to implement the version of ratiw, but as you're saying... 
it's a bit to complicated just to show a table...
If I can ask... does this component suits your needs so far ?

Thanks in advance for your reply,
Johnny

**Comment by jpmurray** [comment]
@jdriesen I went back to ratiw/vue-table, since I knew it well, even if it was complicated ;-)

**Comment by jdriesen** [comment]
@jpmurray Thanks for your quick reply. Highly appreciated. Maybe you're willing to give me a little help... I'm struggling for 3 days now... Do you have an idea where I can find the source-code from this one ? http://vuetable.ratiw.net/examples/bootstrap.html ? (the bootstrap implementation is exactly what I need) 

That would be an amazing help...
Thanks in advance for your reply,

Kind regards,
Johnny

**Comment by jpmurray** [comment]
@jdriesen in their example folder? https://github.com/ratiw/vue-table/tree/master/examples

**Comment by jdriesen** [comment]
Don't know how I've could missed that one. Thanks @jpmurray .

**Comment by lmj0011** [comment]
I decided to give ratiw/vue-table2 another go.

turns out the tutorial got me up to speed fairly quickly.
If you're having trouble with galenyuan/vue-datatable, I would recommend ratiw/vue-table2
https://github.com/ratiw/vuetable-2-tutorial/blob/master/doc/README.md

**Comment by jdriesen** [comment]
Thanks for the info. Gonna give it a another try today.
Rather new to this stuff.

Thanks evrb for the quick responses.

Grtz,
Johnny
